samples/client/petstore/typescript-node/api.ts

```diff
@@ -25,6 +25,20 @@ export class Dog extends Animal {
     "breed": string;
 }
 
+export class FormatTest {
```

Comment:
or you can try doing a rebase on the latest master (which should have fixed the issue in the typescript code generator)